lib/js/src/protocol/SdlProtocolBase.js

```diff
@@ -458,13 +458,12 @@ class SdlProtocolBase {
         const protocolVersion = sdlPacket.getVersion();
         if (protocolVersion >= 5) {
             let rejectedTag = null;
-            let rejectedParams;
             if (serviceType === ServiceType.AUDIO) {
-                rejectedTag = ControlFrameTags.Audio.StartServiceNAK.REJECTED_PARAMS;
+                rejectedTag = ControlFrameTags.Audio.EndServiceNAK.REJECTED_PARAMS;
             }
-            if (rejectedTag === null) {
-                rejectedParams = sdlPacket.getTag(rejectedTag);
-                console.error('Got StartSessionNAK with rejected params', rejectedParams);
+            const rejectedParams = sdlPacket.getTag(rejectedTag);
+            if (rejectedParams !== undefined && rejectedParams !== null && rejectedParams.length > 0) {
```

Comment:
If `rejectedParams` is expected to be an array (as implied by the length check), then `Array.isArray(rejectedParams)` should be used rather than checking for `undefined` and `null`.

Reply:
```suggestion
            if (Array.isArray(rejectedParams) && rejectedParams.length > 0) {
```